examples/grpc/greeter/Makefile

```diff
@@ -0,0 +1,14 @@
+CXXFLAGS ?= -I../../../include
```

Comment:
Yeah, I don't have much experience with cmake, and I don't have non-Linux platforms to test on, so I don't feel qualified to write the `CMakeFile.txt` for the example. I'll leave it as-is for now and someone else can update it later.